test/unit/search/search_test.cpp

```diff
@@ -300,6 +300,13 @@ TYPED_TEST(search_test, search_strategy_all)
         configuration const cfg = max_error{total{1}} | mode{all};
         EXPECT_EQ(uniquify(search("ACGT"_dna4, this->index, cfg)), (hits_result_t{0, 1, 4, 5, 8, 9}));
     }
+
+    { // Test with dynamic mode.
+        mode dynamic_mode{};
```

Comment:
If I remember correctly, in tests we now have to specify `seqan3::` namespace before all types and functions of seqan3?

Reply:
yes, but I won't have this refactoring done for this PR, since it is about a feature. The other thing must be done separately.